projects/1.20/assets/goldelight/goldelight/lang/zh_cn.json

```
@@ -0,0 +1,313 @@
+  "block.goldelight.golden_onions": "金洋葱",
+  "block.goldelight.golden_planks": "金木板",
+  "block.goldelight.golden_potato_crate": "金箱装马铃薯",
+  "block.goldelight.golden_potatoes": "金马铃薯",
+  "block.goldelight.golden_pressure_plate": "金压力板",
+  "block.goldelight.golden_pumpkin": "金南瓜",
+  "block.goldelight.golden_pumpkin_stem": "金南瓜茎",
+  "block.goldelight.golden_rice": "金稻米",
+  "block.goldelight.golden_rice_bag": "金稻米袋",
+  "block.goldelight.golden_rice_bale": "金稻米块",
+  "block.goldelight.golden_rice_panicle": "金稻米穗",
+  "block.goldelight.golden_rice_roll_medley_block": "金寿司拼盘",
+  "block.goldelight.golden_roast_chicken_block": "金烤鸡",
+  "block.goldelight.golden_shepherds_pie_block": "金牧羊人派",
+  "block.goldelight.golden_sign": "金告示牌",
+  "block.goldelight.golden_slab": "金半砖",
+  "block.goldelight.golden_soil": "金泥土",
+  "block.goldelight.golden_soil_farmland": "金耕地",
+  "block.goldelight.golden_stairs": "金楼梯",
+  "block.goldelight.golden_stem": "金菌柄",
+  "block.goldelight.golden_sugar_cane": "金甘蔗",
+  "block.goldelight.golden_tomato_crate": "金箱装番茄",
+  "block.goldelight.golden_tomatoes": "金番茄",
+  "block.goldelight.golden_trapdoor": "金活板门",
+  "block.goldelight.golden_wall_hanging_sign": "金壁挂式标牌",
+  "block.goldelight.golden_wall_sign": "金墙上的告示牌",
+  "block.goldelight.golden_wart": "金下界疣",
+  "block.goldelight.golden_wart_block": "金下界疣块",
+  "block.goldelight.golden_wheat": "金小麦",
+  "block.goldelight.gray_candle_golden_cake": "插上灰色蜡烛的金蛋糕",
+  "block.goldelight.green_candle_golden_cake": "插上绿色蜡烛的金蛋糕",
+  "block.goldelight.groomlight": "Groomlight",
+  "block.goldelight.light_blue_candle_golden_cake": "插上淡蓝色蜡烛的金蛋糕",
+  "block.goldelight.light_gray_candle_golden_cake": "插上淡灰色蜡烛的金蛋糕",
+  "block.goldelight.lime_candle_golden_cake": "插上黄绿色蜡烛的金蛋糕",
+  "block.goldelight.magenta_candle_golden_cake": "插上品红色蜡烛的金蛋糕",
+  "block.goldelight.nether_brick_stove": "下界砖炉灶[WIP]",
+  "block.goldelight.orange_candle_golden_cake": "插上橙色蜡烛的金蛋糕",
+  "block.goldelight.pink_candle_golden_cake": "插上粉红色蜡烛的金蛋糕",
+  "block.goldelight.potted_golden_cactus": "金仙人掌盆栽",
+  "block.goldelight.potted_golden_fungus": "金真菌盆栽",
+  "block.goldelight.potted_golden_mushroom": "金蘑菇盆栽",
+  "block.goldelight.purple_candle_golden_cake": "插上紫色蜡烛的金蛋糕",
+  "block.goldelight.red_candle_golden_cake": "插上红色蜡烛的金蛋糕",
+  "block.goldelight.stripped_golden_hyphae": "去皮金菌核",
+  "block.goldelight.stripped_golden_stem": "去皮金菌柄",
+  "block.goldelight.stuffed_golden_pumpkin_block": "金填馅南瓜",
+  "block.goldelight.sweet_golden_berry_cheesecake": "金甜浆果芝士派",
+  "block.goldelight.white_candle_golden_cake": "插上白色蜡烛的金蛋糕",
+  "block.goldelight.yellow_candle_golden_cake": "插上黄色蜡烛的金蛋糕",
+  "effect.goldelight.immunity": "免疫",
+  "effect.goldelight.immunity.desc": "清除负面效果",
+  "effect.goldelight.immunity.description": "清除负面效果",
+  "effect.goldelight.mildness": "温暖",
+  "effect.goldelight.mildness.desc": "防止被冻结",
+  "effect.goldelight.mildness.description": "防止被冻结",
+  "enchantment.goldelight.midas_touch": "Midas触摸",

```

```suggestion
  "enchantment.goldelight.midas_touch": "迈达斯之触",
```
Midas一般叫弥达斯，不过神话中的用词是迈达斯